k9mail/src/test/java/com/fsck/k9/mailstore/MimePartStreamParserTest.kt

```diff
@@ -0,0 +1,75 @@
+package com.fsck.k9.mailstore
+
+
+import java.io.ByteArrayInputStream
+
+import com.fsck.k9.mail.internet.MimeBodyPart
+import com.fsck.k9.mail.internet.MimeMessage
+import com.fsck.k9.mail.internet.MimeMultipart
+import org.junit.Test
+
+import org.junit.Assert.*
+
+
+class MimePartStreamParserTest {
+    @Test
+    fun innerMessage_DispositionInline() {
+        val msg = MimePartStreamParser.parse(null, ByteArrayInputStream(("From: <x@example.org>\r\n" +
```

Comment:
This can look nicer by using Kotlin's multiline string literal. See e.g. https://github.com/k9mail/k-9/blob/e4467ef959d5dc13d84afb9e49029cc6676fe6f5/k9mail-library/src/test/java/com/fsck/k9/mail/MessageTest.kt#L64-L153